internal/conf/configuration.go

```diff
@@ -406,6 +406,8 @@ type MailerConfiguration struct {
 	EmailValidationServiceHeaders string `json:"email_validation_service_headers" split_words:"true"`
 
 	serviceHeaders map[string][]string `json:"-"`
+
+	MXBlocklist []string `json:"mx_blocklist" split_words:"true"`
```

Comment:
This can only support a handful of hosts, unlikely to be generally useful because of it